core/src/main/java/com/scalar/db/common/CheckedDistributedStorageAdmin.java

```diff
@@ -322,6 +327,11 @@ public void addRawColumnToTable(
 
   @Override
   public Set<String> getNamespaceNames() throws ExecutionException {
+    logger.warn(
```

Comment:
Executing `multiStorageAdmin.getNamespaceNames()` will end up duplicating this message for each `CheckedDistributedStorageAdmin` instance it holds. 
Should something be done about it?

Reply:
Good point. We should avoid duplicate messages. However, I don't have any ideas on how to do that. Do you have any suggestions?

Reply:
@Torch3333 Maybe, at this point, I don't think we have an easy way to do that. So, I think we can keep it as it is for now, and if we come up with a good idea later, we can implement it then. What do you think?

Reply:
I agree, I don't have an idea that does not seem complicated to implement.